packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -238,151 +238,118 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "all actions send to association groups",
+					"label": "All actions send to association groups",
 					"value": 0
 				},
 				{
-					"label": "do not send when switching ON (single click)",
+					"label": "Do not send when switching ON (single click)",
 					"value": 1
 				},
 				{
-					"label": "do not send when switching OFF (single click)",
+					"label": "Do not send when switching OFF (single click)",
 					"value": 2
 				},
 				{
 					"label": "Do not send when changing dimming level",
 					"value": 4
 				},
 				{
-					"label": "do not send on double click",
+					"label": "Do not send on double click",
 					"value": 8
 				},
 				{
-					"label": "send 0xFF value on double click",
+					"label": "Send 0xFF value on double click",
 					"value": 16
 				}
 			]
 		},
 		"25": {
-			"label": "Command frames sent in 4th and 5th assoc grp",
-			"description": "Actions not resulting in sending frames to assoc grp",
+			"label": "Command frames sent in 4th and 5th assoc group (S2)",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 16,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "all actions send to association groups",
+					"label": "All actions send to association groups",
 					"value": 0
 				},
 				{
-					"label": "do not send when switching ON (single click)",
+					"label": "Do not send when switching ON (single click)",
 					"value": 1
 				},
 				{
-					"label": "do not send when switching OFF (single click)",
+					"label": "Do not send when switching OFF (single click)",
 					"value": 2
 				},
 				{
 					"label": "Do not send when changing dimming level",
 					"value": 4
 				},
 				{
-					"label": "do not send on double click",
+					"label": "Do not send on double click",
 					"value": 8
 				},
 				{
-					"label": "send 0xFF value on double click",
+					"label": "Send 0xFF value on double click",
 					"value": 16
 				}
 			]
 		},
 		"26": {
-			"label": "The function of 3-way switch",
-			"description": "Key no. 2 controls the Dimmer additionally",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "3-way switch function for S2 disabled",
-					"value": 0
-				},
-				{
-					"label": "3-way switch function for S2 enabled",
-					"value": 1
-				}
-			]
+			"label": "3-way switch function",
+			"description": "Key S2 also controls the dimmer",
+			"$import": "../templates/master_template.json#base_enable_disable"
 		},
 		"27": {
 			"label": "Associations in Z-Wave network security mode",
-			"description": "Defines commands are sent as secure or non-secure",
+			"description": "Defines commands are sent as secure or non-secure. May be combined, e.g. 1 + 2 = 3 means that 2nd and 3rd are sent as secure",
```

Comment:
This parameter definitely should be split into partial parameters.

Reply:
done